include/kickcat/Frame.h

```diff
@@ -28,15 +28,16 @@ namespace kickcat
         /// \return true if full after adding datagram, false otherwise
         void addDatagram(uint8_t index, enum Command command, uint32_t address, void const* data, uint16_t data_size);
 
-        void clear(); // reset frame context
+        void resetContext();
```

Comment:
Add a comment of what this call is doing (seems even more important now to differentiate clear() and resetContext())